src/common/i18n/locales/en-US.json

```diff
@@ -1822,6 +1831,99 @@
     "update-successful": "Update successful",
     "update-success-message": "Your Recovery email has been updated successfully"
   },
+  "chat": {
+    "page-title": "Conversations",
+    "title": "Chats",
+    "messages": "Messages",
+    "key": "Ecency chat key",
+    "new-message": "New Message",
+    "direct-messages": "Direct messages",
+    "collapse": "Collapse",
+    "expand": "Expand",
+    "scroll-to-bottom": "scroll to Bottom",
+    "joined": "Joined",
+    "followers": "Followers",
+    "start-chat-placeholder": "Start a new message",
+    "search": "Search people",
+    "join-chat": "Join chat",
+    "back": "Back",
+    "no-chat": "No conversation found",
+    "start-chat": "Start chatting",
+    "not-joined": "This user hasn't joined the chat yet",
+    "subscribers": "Subscribers",
+    "invite": "Invite",
+    "send-invite": "Send an invite",
+    "leave:": "Leave",
+    "refresh": "Refresh",
+    "message-warning": "You cannot send messages in this group.",
+    "edit-community-roles": "Edit Community Roles",
+    "add": "Add",
+    "no-admin": "No admin or moderator for this community chat.",
+    "blocked-users": "Blocked Users",
+    "action": "Action",
+    "unblock": "Unblock",
+    "no-locked-user": "No Blocked user yet.",
+    "enter-private-key": "Enter Chat private key",
+    "enter-private-key-detail": "Enter chat private key to import all chats",
+    "close": "Close",
+    "confirm": "Confirm",
+    "error-updating-community": "Error occurred while updating community",
+    "leave": "Leave",
+    "edit-roles": "Edit Roles",
+    "manage-chat-key": "Credentials info",
+    "block": "Block",
+    "communities": "Communities",
+    "dms": "DMs",
+    "import-chat": "Import Chat",
+    "create-new-account": "Create New Account",
+    "loading": "Loading...",
+    "blocked-user-message": "You have been blocked from this community",
+    "chat-joined": "Chat Joined",
+    "start-community-chat": "Start Community Chat",
+    "join-community-chat": "Join Community Chat",
+    "submit": "Submit",
+    "chat-priv-key": "Please, save Ecency key by clicking on download button or scanning QR code to restore your conversation if there will disappeared.",
```

Comment:
`Please, save Chat key by clicking on download button or scanning QR code to restore your conversations.`